pctasks/dataset/pctasks/dataset/workflow.py

```diff
@@ -19,6 +19,21 @@
 from pctasks.ingest.utils import generate_collection_json
 
 
+def task_tags(
+    task_name: str,
+    tags: Optional[Dict[str, str]],
+    task_config: Optional[Dict[str, Any]],
+) -> Optional[Dict[str, str]]:
+
+    task_config_ = {} if task_config is None else task_config
+    tags_ = {} if tags is None else tags
+    merged_tags = tags_ | task_config_.get(task_name, {}).get("tags", {})
```

Comment:
Technically, we support Python 3.8 (https://github.com/microsoft/planetary-computer-tasks/blob/1ecd3ad75931705a9edf102b73507a85b7bd1afa/pctasks/cli/setup.py#L28) and the `|` for merging dicts was added in 3.9 I think.

So maybe

```python
merged_tags = {**tags_, **task_config_.get(task_name, {}).get("tags", {})
```

You'll want to verify that gets the order right.

Reply:
Yep. Fixed. Order is correct.